core/domain/user_services.py

```diff
@@ -314,18 +511,45 @@ def _save_user_settings(user_settings):
 
 
 def is_user_registered(user_id):
+    """Checks if a user is registered with given user_id.
+
+    Args:
+        user_id: str. The user id.
+
+    Returns:
+        A bool based on whether a user is registered with given user_id or not.
```

Comment:
bool. Whether a user with the given user_id has been registered.

(ditto below. Please also drop the "or not" parts.)